images/manageiq-base/container-assets/container_env

```diff
@@ -4,8 +4,45 @@
 
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 
-DATABASE_USERINFO=""
-[[ -n "$DATABASE_USER" && -n "$DATABASE_PASSWORD" ]] && DATABASE_USERINFO="$(ruby --disable-gems -rcgi -e "puts \"#{CGI.escape(ENV['DATABASE_USER'])}:#{CGI.escape(ENV['DATABASE_PASSWORD'])}@\"")"
-DATABASE_QUERY="encoding=utf8&pool=5&wait_timeout=5&sslmode=${DATABASE_SSL_MODE:-prefer}"
-[[ -f /.postgresql/root.crt ]] && DATABASE_QUERY="${DATABASE_QUERY}&sslrootcert=/.postgresql/root.crt"
-export DATABASE_URL="postgresql://${DATABASE_USERINFO}${DATABASE_HOSTNAME:-localhost}:${DATABASE_PORT:-5432}/${DATABASE_NAME:-db_unknown}?${DATABASE_QUERY}"
+if [[ ! -f /var/www/miq/vmdb/config/database.yml ]]; then
```

Comment:
if you don't have a database.yml file then this section runs but if you _do_ have a database.yml file, then database-hostname will never be set.  This means later in the check_svc_status in the entrypoint, you won't have a value.  I think, technically, you need to pull the values from the database.yml if present.

I won't let this hold up the PR, since in reality we never have a database.yml, but then it does bring into question why have this check at all.